src/index.html

```diff
@@ -13,6 +13,52 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
```

Comment:
Use 2 spaces for indentation instead of the tab character to ensure consistent formatting across different environments. This applies to the entire document. Refer to the checklist item 'Use 2 spaces for indentation in your file'.